src/SURL.h

```diff
@@ -0,0 +1,44 @@
+#ifndef _SURL_H_
+#define _SURL_H_
+
+#include <string>
+
+// A data Type to store a short URL and its expire time
+// url name wont be change after the time you created.
+// Example:
+//		SURL surl("https:://www.google.com", "2021/04/19");
+//		// Update the expire time after it is re-created
+//		surl.SetExpireTime(now);
+
+class SURL 
+{
+public:
+
+	// default constructor
+	SURL();
```

Comment:
Is this copyable? If not, please delete copy constructor and copy assignment operator.